extensions/amp-analytics/0.1/events.js

```diff
@@ -81,6 +82,13 @@ const TRACKER_TYPE = Object.freeze({
       return new CustomEventTracker(root);
     },
   },
+  [AnalyticsEventType.AMP_STORY]: {
+    name: AnalyticsEventType.AMP_STORY,
+    allowedFor: ALLOWED_FOR_ALL_ROOT_TYPES.concat(['timer']),
```

Comment:
I don't really think we need the `timer` here. This is used for some events to start/end after a certain amount of time. But I'm also fine leaving it here, doesn't hurt.